node-launchpad/src/bin/tui/main.rs

```diff
@@ -53,12 +54,41 @@ pub struct Cli {
 
     #[command(flatten)]
     pub(crate) peers: PeersArgs,
+
+    /// Print the crate version.
+    #[clap(long)]
+    crate_version: bool,
+
+    /// Print the network protocol version.
+    #[clap(long)]
+    protocol_version: bool,
```

Comment:
As mentioned above, launchpad is not restricted to a certain protocol version, it can use any safenode with any protocol version. So should we remove this arg?

Reply:
This can be removed.